neutral_generation/smart_convert.py

```diff
@@ -0,0 +1,284 @@
+import re
+import math
+import torch
+import spacy
+
+# SpaCy: lowercase is for dependency parser, uppercase is for part-of-speech tagger
+from spacy.symbols import nsubj, aux, conj, neg, VERB, AUX
+from pytorch_pretrained_bert import OpenAIGPTTokenizer, OpenAIGPTModel, OpenAIGPTLMHeadModel
+from constants import *
+
+nlp = spacy.load("en_core_web_sm")
+
+# Load pre-trained model (weights)
+model = OpenAIGPTLMHeadModel.from_pretrained('openai-gpt')
+model.eval()
+# Load pre-trained model tokenizer (vocabulary)
+tokenizer = OpenAIGPTTokenizer.from_pretrained('openai-gpt')
+
+
+def convert(s):
```

Comment:
Let's change 's' to something more descriptive. Avoid single character var names.

Reply:
Got it. Will do.